Documentation/Doxygen/MainPage.dox

```diff
@@ -6,15 +6,14 @@
  *
  * \section intro Introduction
  *
- * Welcome to the National Library of Medicine Insight Segmentation and
- * Registration Toolkit (ITK). ITK is an open-source sofware system to 
+ * Welcome to the Insight Toolkit (ITK). ITK is an open-source sofware system to
```

Comment:
I agree with changing the name, however this paragraph no longer has much of a descriptions of what ITK is. Consider changing to:
> Welcome to the Insight Toolkit (ITK). ITK is an open-source software system for segmentation and registration initially developed to support the Visible Human Project.